lib/ansible/module_utils/urls.py

```diff
@@ -602,7 +602,7 @@ def get_ca_certs(self):
         to_add = False
 
         # Write the dummy ca cert if we are running on Mac OS X
-        if system == 'Darwin':
+        if system == b('Darwin'):
```

Comment:
Fedora 25 Linux, Python-3.5.2:
```
>>> import platform
>>> platform.system()
'Linux'
```

So this seems to be a text string.  If you're getting bytes we should figure out if it's only on Darwin or on some other version of python than 3.5.